src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTreeUI.java

```diff
@@ -3280,6 +3280,9 @@ public Rectangle getNodeDimensions(Object value, int row,
                      expanded, treeModel.isLeaf(value), row,
                      false);
                 if(tree != null) {
+                    for (int i = 0; i < rendererPane.getComponentCount(); i++) {
+                        rendererPane.remove(i);
+                    }
```

Comment:
rendererPane.removeAll()?
```suggestion
                    rendererPane.removeAll();
```